packages/frontend/src/hooks/useQuote.tsx

```diff
@@ -40,6 +41,16 @@ const useQuote = () => {
     setValue(SwapFormKey.ToAmount, formatTokenAmount(quote.toAmount.toString(), toToken?.decimals));
   };
 
+  const handleQuoteFetchError = (error: unknown): void => {
+    if (error instanceof Error) {
+      showToast({
+        text: parseSifiErrorMessage(error.message),
+        type: 'error',
+        toastId: 'quote-error',
```

Comment:
Maybe we can change this to use the fromToken and toToken symbol as well? That way if the user switches to a new token, and there is also no quote available, we'll show a new message to better indicate it's not available for this pair too.

Reply:
![image](https://github.com/sideshiftfi/sifi/assets/112887817/7457b0b9-5ef8-491b-bc2d-4067a8e9b74a)

Reply:
Looks nice! We also need to do it for `toastId` I think. We can set it to `quote-error-${fromToken.symbol}-${fromChain.id}-${toToken.symbol}-${toChain.id}`

That way if the user switches to a different token/chain while they haven't closed the toast, they will get a new error message for the new token pair, as there can only be one toast per toastId in the DOM

Reply:
This reads a bit that ETH/USDT quote is not available at all. But it is about a specific tokens + chains combo.

Reply:
I know, but my requested change is not about that. The toastId should be dynamic. If the user switches token, and the quote is also not available, if the user had not closed the previous quote, they will not get a new detailed error message about the new quote not being available. 

If they were on ETH -> USDT first, and the quote is unavailable they get the toast message. They don't close the toast. They switch to ETH -> USDC and the quote is also not available. Because the previous toast is still visible they won't get a new one, and only the toast with the old token pair is visible.

Reply:
`quote-error-${fromToken.symbol}-${fromChain.id}-${toToken.symbol}-${toChain.id}`

Agree with this, pushing a commit